kwave/utils/io.py

```diff
@@ -277,7 +277,7 @@ def write_attributes(filename: str, file_description: Optional[str] = None, lega
         attributes = {
             h5_literals.FILE_MAJOR_VER_ATT_NAME: h5_literals.HDF_FILE_MAJOR_VERSION,
             h5_literals.FILE_MINOR_VER_ATT_NAME: h5_literals.HDF_FILE_MINOR_VERSION,
-            h5_literals.CREATED_BY_ATT_NAME: f'k-Wave N/A',
+            h5_literals.CREATED_BY_ATT_NAME: 'k-Wave N/A',
```

Comment:
`k-wave-python __version__`?

Reply:
Updated to use `kwave.VERSION`, good point!